cloudzero/README.md

```diff
@@ -0,0 +1,45 @@
+# Agent Check: CloudZero
+
+## Overview
+
+CloudZero helps engineering teams build cost-effective software. Through its innovative platform, CloudZero allocates 100% of customers' cloud, PaaS, and SaaS spend—regardless of their tagging quality—and presents it in a single unified view. It then combines hourly spend data with business- and system-level telemetry to put cloud data in a business context through precise unit cost metrics like cost per customer, per product, per feature, per team, and more. CloudZero's AI-powered anomaly detection alerts engineers to abnormal spend events by pointing them directly to the affected infrastructure, and promotes engineering engagement in cloud cost management.
```

Comment:
Per the GitHub check, can you replace any non-ASCII characters in here?  (they usually happen when pasting from something like Microsoft Word).